src/features/movies/ui/movie-list/index.js

```
@@ -24,34 +25,41 @@ export const updateMovieList = (movies) => {
 
 const onClickLoadButton = async (onLoadMore) => {
   const movieList = document.querySelector(".thumbnail-list");
+  hiddenFallbackView();
```

hiddenFallbackView는 명사로 읽혀서 `hide-*`를 쓰면 어떨까 싶네요!